cmd/test-image/tcp/tcp.go

```diff
@@ -0,0 +1,46 @@
+package tcp
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/docker/libnetwork/cmd/test-image/server"
+	"github.com/sirupsen/logrus"
+)
+
+var basePaths = map[string]server.HTTPHandlerFunc{
+	"/tcp_ping": tcp_ping,
+	"/tcp_pong": tcp_pong,
+}
+
+func Init(s *server.TestServer) {
+	s.RegisterHandler(basePaths)
+}
+
+func tcp_ping(s *server.TestServer, w http.ResponseWriter, r *http.Request) {
+	logrus.Info("tcp_ping")
```

Comment:
I'm probably a bit old-fashioned here, but this is less about TCP and more about HTTP.  I was expecting operations that were a bit lower level in this module.

Reply:
yep actually addressed in the new version